The purpose of this patch is to backport a fix that is already on upstream 
trunk/, the reason is that if rename fails we don't want error messages 
(see http://bugs.debian.org/555932)

--- a/src/item-lib.c
+++ b/src/item-lib.c
@@ -242,7 +242,7 @@
 
 if (rename(new,file) == -1)
    {
-   CfOut(cf_error,"rename","Error while renaming %s\n",file);
+   CfOut(cf_inform,"rename","Error while renaming %s\n",file);
    return false;
    }       
 
